src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -43,6 +43,7 @@
         <service id="form.type_guesser.validator" class="Symfony\Component\Form\Extension\Validator\ValidatorTypeGuesser">
             <tag name="form.type_guesser" />
             <argument type="service" id="validator.mapping.class_metadata_factory" />
+            <deprecated>"The \"%service_id%\" service is deprecated since Symfony 3.1 and will be removed in 4.0."</deprecated>
```

Comment:
@HeahDude if you remove this service, type guessing will not work properly anymore (it won't read validator metadata anymore).
Using the SE alone would not show it, as there is nothing in it relying on this feature (you have to write code using forms).